templates/cloudflare-workers/vite.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from "vite";
+import {
+  vitePlugin as remix,
+  cloudflareDevProxyVitePlugin,
+} from "@remix-run/dev";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+export default defineConfig((env) => ({
+  plugins: [cloudflareDevProxyVitePlugin(), remix(), tsconfigPaths()],
+  ...(env.isSsrBuild && {
+    resolve: {
+      conditions: ["workerd", "worker", "browser"],
+      mainFields: ["browser", "module", "main"],
+    },
+    build: {
+      minify: true,
```

Comment:
Why is `minify: true` needed for CF workers?

Reply:
This was ported from the old template. I guess the idea is to keep the file size to a minimum.